app/models/manageiq/providers/hawkular/concerns/server_operations.rb

```diff
@@ -0,0 +1,87 @@
+module ManageIQ::Providers
+  module Hawkular
+    module ServerOperations
```

Comment:
Actually, having concerns as a different folder, but without namespacing it is the rails default (try creating a new rails application and check the `app/models` folder.